src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -430,4 +430,24 @@ public Response removeDacDaaRelationship(
       return createExceptionResponse(e);
     }
   }
+
+  @POST
+  @RolesAllowed({ADMIN, CHAIRPERSON})
+  @Path("{dacId}/upload/{oldDaaId}/{newDaaName}")
+  public Response sendNewDaaMessage(
+      @Auth AuthUser authUser,
+      @PathParam("dacId") Integer dacId,
+      @PathParam("oldDaaId") Integer oldDaaId,
+      @PathParam("newDaaName") String newDaaName
+      ) {
+    try {
+      User user = userService.findUserByEmail(authUser.getEmail());
+      Dac dac = dacService.findById(dacId);
+      String dacName = dac.getName();
+      daaService.sendNewDaaEmails(user, oldDaaId, dacName, newDaaName);
```

Comment:
It would be good to try and fail fast here if the old or new DAA were not found

Reply:
I can move the logic of checking if the old or new DAA exist from the service class to the resource class!